charts/seed-operatingsystemconfig/downloader/templates/scripts/_gardener-user.sh

```diff
@@ -0,0 +1,11 @@
+{{- define "seed-operatingsystemconfig.gardener-user" -}}
+#!/bin/bash -eu
+
+id gardener || useradd gardener -mU
+mkdir -p /home/gardener/.ssh
+find /home -name authorized_keys -not -path "/home/gardener/*" -exec cp -f {} /home/gardener/.ssh/authorized_keys \;
```

Comment:
Hey, @vpnachev , does this also work on alicloud?

Reply:
just for reference, https://github.com/gardener/gardener-extensions/pull/355